esti/system_test.go

```diff
@@ -75,10 +91,7 @@ func generateUniqueRepositoryName() string {
 }
 
 func generateUniqueStorageNamespace(repoName string) string {
-	storageNamespace := viper.GetString("storage_namespace")
-	if !strings.HasSuffix(storageNamespace, "/") {
-		storageNamespace += "/"
-	}
+	storageNamespace := fmt.Sprintf("%s/%s/", viper.GetString("storage_namespace"), xid.New().String())
```

Comment:
You removed the optional '/' concat - if storage namespace will include '/' you will add '//' which means something else.